scripts/delayed.js

```diff
@@ -150,21 +150,19 @@ function loadPrivacy() {
 
   // Configure Privacy
   window.fedsConfig = {
+    locale: getLanguage(), // use supported locale
+    disableTarget: true, // this will bypass Target A/B tests
+    content: {
+      experience: 'privacy', // authored experience with Privacy component
+    },
     privacy: {
-      otDomainId: getOtDomainId(),
+      otDomainId: getOtDomainId(), // your OneTrust domain ID - see list of domains
+      footerLinkSelector: '[href="https://www.adobe.com/#openPrivacy"]', // CSS selector that will open the privacy modal
```

Comment:
this should work even if the DOM node is not available at the time privacy loads. Make sure to click on "cookie preferences" in the footer and test.